src/routing/mod.rs

```diff
@@ -256,7 +253,7 @@ impl<S> Router<S> {
     /// routes.
     pub fn boxed<ReqBody, ResBody>(self) -> Router<BoxRoute<ReqBody, S::Error>>
     where
-        S: Service<Request<ReqBody>, Response = Response<ResBody>> + Send + 'static,
+        S: Service<Request<ReqBody>, Response = Response<ResBody>> + Clone + Send + Sync + 'static,
```

Comment:
Adding these bounds is a breaking change. It was also required to make https://github.com/tokio-rs/axum/pull/270 work.